<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmSettingsDialog</class>
 <widget class="QDialog" name="AlarmSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTimeEdit" name="timeEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineLabel"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Repeat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboRepeat"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupDays">
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Specific days</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chkMon">
           <property name="text">
            <string>Mon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkTue">
           <property name="text">
            <string>Tue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkWed">
           <property name="text">
            <string>Wed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkThu">
           <property name="text">
            <string>Thu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkFri">
           <property name="text">
            <string>Fri</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSat">
           <property name="text">
            <string>Sat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSun">
           <property name="text">
            <string>Sun</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Sound</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSound"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnBrowseSound">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSnooze">
        <property name="text">
         <string>Snooze</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
